prow/scripts/cluster-integration/control-plane-gke-integration.sh

```diff
@@ -274,6 +274,15 @@ data:
   auditlog-config-path: "/path"
   auditlog-security-path: "/path"
   auditlog-tenant: "tnt"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cls-config
```

Comment:
Could you please rename it to `kcp-cls-config` in order to stick to the convention? @ebensom will also rename the real secret

Reply:
fixed.